roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -172,9 +191,11 @@ spec:
           initialDelaySeconds: 30
           timeoutSeconds: 30
       volumes:
+{% if not dashboard_use_custom_certs %}
```

Comment:
Can you revert all of the changes in this if block? `auto-generate-certificates` is compatible with the secret volume